src/main/java/me/totalfreedom/totalfreedommod/rank/RankManager.java

```diff
@@ -152,13 +158,13 @@ public void onPlayerJoin(PlayerJoinEvent event)
             player.getInventory().clear();
             player.setOp(false);
             player.setGameMode(GameMode.SURVIVAL);
-            plugin.pl.getPlayer(player).getFreezeData().setFrozen(true);
+            plugin.pl.getPlayer(player).setEditBlocked(true);
```

Comment:
This PR will not be merged with this change to the file. If there is a bug with freezing it needs to be raised as a bug and fixed before this can be merged now.